web/template/donor/your_address.gohtml

```diff
@@ -1,16 +1,18 @@
 {{ template "page" . }}
 
-{{ define "pageTitle" }}{{ tr .App "whatIsYourAddress" }}{{ end }}
+{{ define "pageTitle" }}{{ tr .App "yourAddress" }}{{ end }}
 
 {{ define "main" }}
   <div class="govuk-grid-row">
     <div class="govuk-grid-column-two-thirds">
       <form novalidate method="post">
         <fieldset class="govuk-fieldset">
           <legend class="govuk-fieldset__legend govuk-fieldset__legend--xl">
-            <h1 class="govuk-fieldset__heading">{{ tr .App "whatIsYourAddress" }}</h1>
+            <h1 class="govuk-fieldset__heading">{{ tr .App "yourAddress" }}</h1>
           </legend>
 
+          {{ template "warning" (content .App "yourAddressWarning") }}
```

Comment:
this should only appear when going from the change link on "make a new LPA", right?